Added Geany cheat sheet

tagawa opened the pull request:
**What does your Instant Answer do?**
Shows a list of keyboard shortcuts for the Geany code editor (Linux only).

**What is the data source for your Instant Answer? (Provide a link if possible)**
http://wiki.geany.org/howtos/sorted_keyboard_shortcuts

**Why did you choose this data source?**
This is the definitive wiki for Geany.

**Are there any other alternative (better) data sources?**
Not that I'm aware of.

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Developers using Linux.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
https://duck.co/ia/view/geany_cheat_sheet

**Are you having any problems? Do you need our help with anything?**
Nope.

**Where did you hear about DuckDuckHack? (For first time contributors)**


**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
![geany-cheatsheet](https://cloud.githubusercontent.com/assets/94173/9562362/af642cee-4ea5-11e5-8932-d231cae67c33.png)

-----
IA Page: https://duck.co/ia/view/geany_cheat_sheet

mintsoft commented:
@tagawa lgtm, but a little text-heavy? Can we shorten some of the descriptions somewhat?

mintsoft commented:
Also hello to a fellow vivaldi user!

moollaza commented:
Code LGTM :+1: I think we may want to slip down the text. @tagawa if the source has the same info then I think it's OK to be less verbose. We could add to the description that things are explained in more detail at the source?

jagtalon commented:
@tagawa @moollaza I think I agree with that. `Go to Start of Display Line` instead of `Go to Start of Display Line: Move the caret to the start of the display line. This is useful when you use line wrapping and want to jump to the start of the wrapped, virtual line, not the real start of the whole line. If the line is not wrapped, it behaves like Go to Start of Line.` because it looks like a wall of text. :)

jagtalon commented on share/goodie/cheat_sheets/json/geany.json:
```diff
@@ -0,0 +1,303 @@
+            "key": "F5"
+        }, {
+            "val": "Show macro list: Shows a list of available macros and variables in the workspace.",
+            "key": "[Ctrl] [Return"
```
Missing `]`?

jagtalon commented:
Ok I guess there are some exceptions such as the two entries with `Tab`: "Complete snippet: If you type a construct like ..."

tagawa commented:
Thanks for the suggestions &mdash; I agree.

I've removed a lot of the superfluous text and fixed the missing closing bracket.

![geany-cheatsheet](https://cloud.githubusercontent.com/assets/94173/9563541/0a58b356-4ec2-11e5-867c-825729e5a034.png)

mintsoft commented:
@tagawa That looks much better!

jagtalon commented:
@tagawa that looks sweet!

mintsoft commented:
@jagtalon @moollaza  this LGTM, any objection to it being merged?

jagtalon commented:
It looks good to me @mintsoft. Merge away!

tagawa commented:
Thank you all! Same day merge &mdash; you folks are on :fire:
